scheduler/src/cook/mesos/api.clj

```diff
@@ -2242,7 +2244,12 @@
                  (timers/time!
                    (timers/timer ["cook-scheduler" "handler" "list-endpoint-duration"])
                    (let [job-uuids (list-jobs db false ctx)]
-                     (mapv (partial fetch-job-map db framework-id) job-uuids))))))
+                     (histograms/update!
+                       (histograms/histogram ["cook-scheduler" "handler" "list-endpoint-response-count"])
+                       (count job-uuids))
```

Comment:
Why are we doing the `(count job-uuids)`?

Reply:
I'm histogramming the response sizes (as counted by # jobs in the response) as a metric. E.g., to discover that 10% of our queries are returning 10000 jobs in the response. :)

Reply:
We already have `cook-scheduler.cook-mesos.api.list-response-job-count.*` - doesn't that have what you want?

Reply:
I just noticed that, sitting in list-jobs, about 15 minutes ago and am updating the review with this nuked.